packages/app-tests/e2e/test-document-flow.spec.ts

```diff
@@ -0,0 +1,79 @@
+import { expect } from '@playwright/test';
+
+import { test } from '../fixtures/fixtures';
+
+test.use({ storageState: { cookies: [], origins: [] } });
+
+const testSigners = [
+  { email: 'signer1@gmail.com', name: 'signer 1' },
+  { email: 'signer2@gmail.com', name: 'signer 2' },
+  { email: 'signer3@gmail.com', name: 'signer 3' },
+];
+
+test.describe('test document flow', async () => {
+  test.beforeEach(async ({ users }) => {
+    const user = await users.create();
+    await user.apiLogin();
+  });
+  test.afterEach(async ({ users }) => users.deleteAll());
+
+  test('user can upload documents', async ({ page, samplePdf }) => {
+    await page.goto('/documents');
+    await expect(page).toHaveURL('/documents');
+
+    await page.getByTestId('document-dropzone').setInputFiles({
+      buffer: samplePdf,
+      mimeType: 'application/pdf',
+      name: 'test.pdf',
```

Comment:
Is there an existing `test.pdf` file?

Reply:
No it's just a placeholder name

Reply:
why not use the pdf name as the name? instead of using `test.pdf`?

Reply:
used pdf's name as the name